README.md

```diff
@@ -66,5 +68,50 @@ fetcher.on("consentString", data => {
 });
 ```
 
+### Vendor consents
+It's possible to fetch individual vendor consent flags using this library:
+
+```javascript
+import { ConsentStringFetcher } from "get-consent";
+
+const fetcher = new ConsentStringFetcher(myWin);
+
+fetcher.on("vendorConsentsData", data => {
+    // {
```

Comment:
Wondering a bit, why is this part commented, could it be removed or is it used as a reference for something?